nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const router = express.Router();
+
+const reservations = require("./../data/reservations.json");
+
+// router.get("/", async (request, response) => {
```

Comment:
Avoid leaving comments that are unused/not useful, your code will be cleaner and easier to read